modules/cnv-add-nic-to-vm.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-edit-vms.adoc
+// * cnv/cnv_users_guide/cnv-editing-vm-template
+//
+// Establishing conditionals so content can be re-used for editing VMs
+// and VM templates.
+
+ifeval::["{context}" == "cnv-add-nic-to-vm"]
+:cnv-vm-add-nic:
+:object: virtual machine
+:object-gui: Virtual Machines
+endif::[]
+
+ifeval::["{context}" == "cnv-add-nic-to-vm-template"]
```

Comment:
And this one needs to be `cnv-editing-vm-template`

Reply:
Done.